packaging/flavorgen/flavors/clusterclass_helpers.go

```diff
@@ -0,0 +1,151 @@
+/*
```

Comment:
nit: Since this file has only ClusterClass patches, consider moving it to a `patches.go` file. Not if it requires significant rework.